packages/components/style.css

```diff
@@ -5,6 +5,8 @@
 @import 'tailwindcss/components' layer(l-components);
 @import 'tailwindcss/utilities';
 
+@import './dist/index.css';
```

Comment:
are you bundling some style now?

Reply:
Yeah, I have this file with color schemes for the MarketplaceSearch and overrides for `nextra` Tabs inside them.

- Tabs can't be styled at this moment. This is something we could move to Tailwind, but we'd have to change Tabs in Nextra to accept `className` first.
- The color schemes were a pain to write in Tailwind as it's just custom classes for CSS properties. Gonna be better for the bundle to have them in a CSS file.

<img width="585" alt="image" src="https://github.com/user-attachments/assets/27a4dbc5-0d69-4100-831b-af72a851378a">